snyk-monitor/templates/deployment.yaml

```diff
@@ -172,6 +176,13 @@ spec:
             name: {{ .Values.defaultWorkloadPoliciesMap }}
             {{- end }}
             optional: true
+        {{- if .Values.volumes.projected.serviceAccountToken }}
+        - name: token-vol
+          projected:
+            sources:
+            - serviceAccountToken:
+                path: token  
+        {{- end }}
```

Comment:
Hey @tektite-io I was wondering if this volume also needs to be mounted with a `volumeMount`? The example in the docs both declares the `volume` and a `volumeMount` - https://kubernetes.io/docs/concepts/storage/volumes/#example-configuration-secret-downwardapi-configmap
And if we have to mount this volume, where should it go?

Also, it looks like token expires by default after 1 hour according to the docs. How should we handle this?